src/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
```

Comment:
We don't need to add empty lines between parent and child elements.
```suggestion
<html lang="en">
  <head>
```